src/structures/APIMessage.js

```diff
@@ -63,6 +64,16 @@ class APIMessage {
     return this.target instanceof User || this.target instanceof GuildMember;
   }
 
+  /**
+   * Whether or not the target is a message
+   * @type {boolean}
+   * @readonly
+   */
+  get isMessage() {
+    const Message = require('./Message');
```

Comment:
You can replace the require-statement from here to the top of the file since not only it is considered good practice but the code looks clean and organized.

Reply:
Actually no, as that causes circular requires.